subxt/src/tx/tx_payload.rs

```diff
@@ -65,10 +65,6 @@ pub struct Payload<CallData> {
     validation_hash: Option<[u8; 32]>,
 }
 
-/// A boxed transaction payload.
-// Dev Note: Arc used to enable easy cloning (given that we can't have dyn Clone).
-pub type BoxedPayload = Payload<Arc<dyn EncodeAsFields + Send + Sync + 'static>>;
```

Comment:
Note: This was removed, because `EncodeAsFields` is not object safe anymore and in the public element room for subxt noone seemed to care about it not being supported anymore.